roles/lnls-ans-role-network/README.md

```diff
@@ -14,9 +14,9 @@ This Ansible role configures some network parameters for Sirius Light Source con
 ---
 # Package versions for Debia. Leave empty for the latest
 pkg_network_manager_version: ""
```

Comment:
isn't necessary to update this one to `pkg_version_network_manager` as well?

Reply:
yes, @anacso17, thanks. i just comitted a fix.

@lerwys , there are a number of undefined package version variables in this role's _defaults/main.yml_:

> pkg_version_network_manager_glib: ""
> pkg_version_libnm_qt_devel: ""
> pkg_version_nm_connection_editor: ""
> pkg_version_libsemanage_python: ""
> pkg_version_policycoreutils_python: ""
> 

 shouldn't we define specific versions in _group_vars/all_ ?